pkg/deployments/scaler.go

```diff
@@ -86,9 +86,9 @@ func (s *scaler) compareScales(current, desired int32) {
 
 	log.Printf("Comparing scales, current: %v, desired: %v", s.currentScale, s.desiredScale)
 
-	if s.desiredScale > s.currentScale {
+	if s.desiredScale > s.currentScale { //nolint:gocritic
```

Comment:
the linter suggests a `switch` statement instead. I kind of agree but this is personal preference